service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/DocumentUploadedEventHandler.java

```diff
@@ -323,6 +359,49 @@ private List<Element<SupportingEvidenceBundle>> getEvidenceBundle(CaseData caseD
         }
     }
 
+    private List<FurtherDocument> getFurtherDocumentsForMainApplication(CaseData caseData) {
+        List<FurtherDocument> ret = new ArrayList<>();
+
+        Function<Element<? extends FurtherDocument>, FurtherDocument> mapper = (ad) -> (FurtherDocument) ad.getValue();
+
+        // Further application documents - for example the SWET or care plan
+        List<Element<ApplicationDocument>> applicationDocuments = defaultIfNull(
+            caseData.getApplicationDocuments(), List.of());
+        ret.addAll(applicationDocuments.stream().map(mapper).collect(toList()));
```

Comment:
Do we need to define mapper above? Could we maybe just do map(Element::getValue) instead?